### PR TITLE
Stake Calculation Fix

### PR DESCRIPTION
**Pull request by sherlockcoin**
Stake Calculation Fix
Checkpointed
Version Bumped
Visual Fixes

### CODE REVIEW

**Comment by sherlockcoin** [comment]
Updated

File: src/main.cpp [hunk 1]
```diff
@@ -1017,7 +1017,7 @@ int64_t GetProofOfWorkReward(int64_t nFees)
 // miner's coin stake reward based on coin age spent (coin-days)
 int64_t GetProofOfStakeReward(int64_t nCoinAge)
 {
-    int64_t nSubsidy = nCoinAge * COIN_YEAR_REWARD * 33 / (365 * 33 + 8);
+    int64_t nSubsidy = nCoinAge * COIN_YEAR_REWARD * 33.0 / (365 * 33 + 8);
```
Comment (Laser9un) [inline src/main.cpp]: How will this fix a "Stake Calculation" in the system with a reward of 0% per year?

File: src/qt/forms/aboutdialog.ui [hunk 2]
```diff
@@ -66,7 +66,7 @@
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
-          <string notr="true">0.3.666-beta</string>
```
Comment (Laser9un) [inline src/qt/forms/aboutdialog.ui]: Where is this string currently visible?

**Comment by Laser9un** [comment]
The main change was merged: https://github.com/42-coin/42/commit/a3ba5db411e4dfd2889fbbad5782a43c11161142

**Pull request closed without merging**